### DISCUSSION TITLE
Database migration

### DISCUSSION THREAD

**Discussion by sebplorenz** [International-Data-Spaces-Association/DataspaceConnector#113] [Q&A] [answered]
Hi, I successfully updated from version 6.4.0 to 7.0.0 with the following settings:

```
spring.flyway.enabled=true
spring.flyway.baseline-on-migrate=true
spring.flyway.baseline-version=<your-last-connector-version>
```
The documentation says:

> The migration should remain enabled after it has been performed once, as new connector versions might introduce changes to the database which require another migration.

Since I am on version 7.0.0 now should I keep the baseline-on-migrate enabled and set the baseline-version to 7.0.0? Or can I remove the baseline settings and set only spring.flyway.enabled=true ?

**Comment by ronjaquensel** [comment, marked as answer]
Hi, you should leave the `baseline-on-migrate` setting enabled. You can set the baseline to `7.0.0`, but after the migration has been performed once, Flyway puts information about it in the database. Therefore, even if you leave the baseline at e.g. `6.4.0`, you would see the following logs starting the connector again:
```
2022-01-28T08:51:24,332 [restartedMain] INFO - Successfully validated 14 migrations (execution time 00:00.027s)
2022-01-28T08:51:24,343 [restartedMain] INFO - Current version of schema "public": 7.0.0
2022-01-28T08:51:24,344 [restartedMain] INFO - Schema "public" is up to date. No migration necessary.
```
